### DISCUSSION TITLE
Can it support Aliyun OSS?

### DISCUSSION THREAD

**Discussion by LeeMo2K10** [duckdb/duckdb#9465] [Ideas]
DuckDB 看是支持S3， 可以支持aliyun oss  么？

**Comment by Mause** [comment]
Have you tried it out to see if it works?

For our future reference: Aliyun appears to be Alibaba's S3 clone

**Reply by LeeMo2K10** [reply]
Error: IO Error: No files found that match the pattern "oss://xxx/tmp/integral_wall/br.csv"

**Reply by LeeMo2K10** [reply]
SET s3_endpoint='oss-us-east-1.aliyuncs.com';
SET s3_region='us-east-1';
SET s3_access_key_id='xxxx';
SET s3_secret_access_key='xxxx';
select * from read_csv_auto('s3://xxx/tmp/part-00000') limit 10;       
可以了， very nice
